src/makers/example/index.ts

```diff
@@ -4,6 +4,6 @@ import hotspots from './hotspots'
 export default {
   antennas,
   hotspots,
-  id: 99999, // TODO: Get maker id from onboarding server
-  supportEmail: 'support@examplemaker.com',
+  id: 99999, // TODO: Get maker id from onboarding server after creation
```

Comment:
@vpetersson after you run the node create-maker task, we need to update this with the corresponding ID